tools/scripts/ignore-build-step.js

```diff
@@ -0,0 +1,74 @@
+const owner = process.env.VERCEL_GIT_REPO_OWNER
+const repo = process.env.VERCEL_GIT_REPO_SLUG
+const pullRequestId = process.env.VERCEL_GIT_PULL_REQUEST_ID
+const commitRef = process.env.VERCEL_GIT_COMMIT_REF
+
+const APP_ARGV = '--app='
+const appName = (() => {
+  const argv = process.argv.find((arg) => arg.startsWith(APP_ARGV))
+  return argv ? argv.slice(APP_ARGV.length) : undefined
+})()
+
+const PREVIEW_IGNORE_BRANCHES = ['main', 'configuration', 'release-please--branches--main']
```

Comment:
Where is the `release-please--branches--main` used?
Don't think I ever saw that.

Reply:
here for example https://github.com/cowprotocol/cowswap/pull/5254

Reply:
Ah, there!